docs/products/flink/concepts/managed-service-features.rst

```diff
@@ -0,0 +1,44 @@
+Aiven for Apache Flink® managed service features
+================================================
+
+As a fully managed distributed data stream processing platform, deployable in the cloud of your choice, some of the key features of a managed Flink service include the following:
+
+Cluster deployment mode
+-----------------------
+Aiven for Apache Flink® is configured to use the `HashMap state backend <https://ci.apache.org/projects/flink/flink-docs-stable/api/java/org/apache/flink/runtime/state/hashmap/HashMapStateBackend.html>`_. This means that the `state <https://ci.apache.org/projects/flink/flink-docs-release-1.15/docs/concepts/stateful-stream-processing/#what-is-state>`_ is stored in memory, which can impact the performance of jobs that require keeping a very large state. We recommend you provision your platform accordingly.
+
+The Flink cluster executes applications in `session mode <https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/deployment/overview/#session-mode>`_ so you can deploy multiple Flink jobs on the same cluster, thus effectively utilizing the available resources.
```

Comment:
same here, the link points to version 1.13, we should either fix a newer version or use the master version